src/ethereum_test_exceptions/exception_mapper.py

```diff
@@ -48,14 +48,19 @@ def __init__(self) -> None:
             exception: re.compile(message) for exception, message in self.mapping_regex.items()
         }
 
-    def message_to_exception(self, exception_string: str) -> ExceptionBase | UndefinedException:
+    def message_to_exception(
+        self, exception_string: str
+    ) -> List[ExceptionBase] | UndefinedException:
         """Match a formatted string to an exception."""
+        exceptions: List[ExceptionBase] = []
         for exception, substring in self.mapping_substring.items():
             if substring in exception_string:
```

Comment:
if there are now regex strings should be accounted here?

Reply:
Yes in line 56 we go over all regex patterns to try to match any.